sphinx_immaterial/apidoc/python/style_default_values_as_code.py

```diff
@@ -12,9 +10,9 @@ def _monkey_patch_python_parse_arglist():
     orig_parse_arglist = sphinx.domains.python._parse_arglist
 
     def parse_arglist(
-        arglist: str, env: Optional[sphinx.environment.BuildEnvironment] = None
+        arglist: str, *args, **kwargs
```

Comment:
This is the same solution I was going to suggest for #265. So, I think this should close #265, but I was wondering if their multi-line approach could be coupled with this theme's multi-line formatting of signatures. Probably better to keep them separate due to the options this theme has for clang-format on C family domains.